day_two_guide/topics/etcd_backup.adoc

```diff
@@ -5,57 +5,19 @@ Module included in the following assemblies:
 
```

Comment:
It took me a little bit to figure out why you took some info out of the below and create a new module about it, then I saw the "con_" prefix and figured it out... Can I suggest updating this file to be "proc_etcd_backup.adoc" to be consistent?

Reply:
If you don't mind, sure!